src/api/dependency-discovery/src/github.js

```diff
@@ -0,0 +1,60 @@
+const { request } = require('@octokit/request');
+
+const hasGitHubExpired = (dateTime) => Date.now() - dateTime > 60 * 60 * 1000;
+
+const gitHubInformationRequestor = {};
+
+async function constructGitHubRequest(packageName, gitHubUrl) {
+  const labelsToSearchFor = ['hacktoberfest', 'good first issue', 'help wanted'];
```

Comment:
I wonder if we are able to add more labels for this, or the GitHub request only asks for those 3 labels?

Reply:
We can easily add more but these are most popular beginner friendly labels